lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +157,35 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
+
+        link = article_bs.get('href')
+        if (link is not None) and
         pass
 
     def find_articles(self) -> None:
         """
         Finds articles
         """
+        for link_bs in self.config._extract_config_content().get_seed_urls:
+            link = link_bs.get('href')
+            if link is None:  # no ==
+                continue
+            elif link[0] == 'n' and link[1] == 'e' and link[2] == 'w' and link.count('#') == 0 and link.count('?') == 0:
```

Comment:
can you suggest other ways of getting links? maybe use regular expressions or check specific string is included or not

Reply:
fixed